packages/venia-ui/lib/components/CartPage/cartPage.js

```diff
@@ -8,9 +9,13 @@ import PriceSummary from './PriceSummary';
 import ProductListing from './ProductListing';
 import { mergeClasses } from '../../classify';
 import defaultClasses from './cartPage.css';
+import { CartPageFragment } from './cartPageFragments';
```

Comment:
I would support this overfetch if it prevented network calls in sub-components OR if you drilled the data down. Via chat you confirmed the first is not true; lets use a slimmer fragment that only grabs the data this component needs (total_quantity).